src/Orc.Controls.Example/ViewModels/CalloutViewModel.cs

```diff
@@ -1,18 +1,26 @@
 ﻿namespace Orc.Controls.Example.ViewModels
 {
+    using System;
     using System.Threading.Tasks;
     using System.Windows;
+    using System.Windows.Threading;
     using Catel;
     using Catel.MVVM;
 
     public class CalloutViewModel : ViewModelBase
     {
+        private readonly DispatcherTimer _showCalloutDispatcherTimer = new DispatcherTimer();
+
         public CalloutViewModel(ICalloutManager calloutManager)
         {
             Argument.IsNotNull(() => calloutManager);
 
             CalloutManager = calloutManager;
             OpenCallout = new TaskCommand<object>(OpenCalloutExecuteAsync);
+            ToggleShowRepeatedly = new TaskCommand(OnShowRepeatedlyExecuteAsync);
+
+            _showCalloutDispatcherTimer.Interval = TimeSpan.FromSeconds(3);
```

Comment:
What's this feature? Is it for debugging, or ?

Reply:
It's for example application. Toggle button repeadetly showing callouts every 3 seconds if you want to test interaction of popus behavior with other windows